views/partials/_contact-panel.njk

```diff
@@ -50,14 +50,14 @@
   <p class="govuk-body app-contact-panel__body">
     If you’ve got a question about the GOV.UK Design System you can contact the team:
   </p>
-  <ul class="govuk-list govuk-list--bullet">
+  <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-0">
```

Comment:
If this content changed the back-to-top link would be out alignment. I wonder whether it might be better to raise `.app-back-to-top` by adding `bottom` pixels to it, and leave a comment to say in SCSS to say the bottom pixels track the height of the contact panel. It wouldn't stop the misalignment if the content changed but at least there would be a comment about it in the code 🤔

Reply:
I think it's easier to have logic that the last bits of content do not have a bottom margin since paragraphs and contact-panels (and other stuff) have different margins.

Reply:
Okay that let's go with this, thanks 👍